src/routes/documents/[id]-[slug]/sidebar/Actions.svelte

```diff
@@ -131,29 +110,44 @@
 
   {#if document.edit_access}
     <!-- TODO: Processing component -->
-    <SidebarItem disabled={!modal || document.status === "nofile"}>
+    <SidebarItem disabled={document.status === "nofile"}>
       {#if document.status !== "success"}
         {$_("status.status")}:
         {$_(`status.${document.status}.title`)}
       {/if}
-      <Action on:click={openReprocess}>
+      <Action on:click={() => reprocessOpen = true}>
         <IssueReopened16 />
         {$_("sidebar.reprocess")}
       </Action>
+      {#if reprocessOpen}
+      <Portal>
+        <Modal on:close={() => reprocessOpen = false}>
+          <h1 slot="title">{$_("dialogReprocessDialog.title")}</h1>
+          <Reprocess documents={[document]} on:close={() => reprocessOpen = false} />
+        </Modal>
+      </Portal>
+      {/if}
     </SidebarItem>
   {/if}
 
   {#if document.edit_access}
     <SidebarItem>
       <Action
-        on:click={openDelete}
-        disabled={!modal}
-        --fill="var(--caution)"
-        --color="var(--caution)"
+        on:click={() => deleteOpen = true}
+        --fill="var(--orange-3)"
```

Comment:
Should this just be the `--caution` color? Right now both `--caution` and `--danger` are the same red, but one could be orange if we wanted. (Not going to hold up this PR for this question.)

Reply:
Agree on both points. Caution works great as an alias for orange, and we can change that separately.